superdesk/macros/set_default_template_metadata.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import superdesk
+from flask_babel import lazy_gettext
+from superdesk import get_resource_service
+
+fields_to_exclude = superdesk.app.config.get('DEFAULT_TEMPLATE_METADATA_MACRO_EXCLUDE')
+fields_to_override = superdesk.app.config.get('DEFAULT_TEMPLATE_METADATA_MACRO_OVERRIDE')
```

Comment:
you should only resolve this when macro is called, otherwise the `app.config` might not be loaded yet